src/app/components/search/search-api.service.ts

```diff
@@ -4,8 +4,8 @@ import { Observable, map } from 'rxjs';
 import { DtoTransformService } from 'src/app/services/dto-transform.service';
 import { MediaItemDTOWithType } from 'src/app/types/DTO/media-item-dto';
 import { MediaListResponse } from 'src/app/types/media-list-response';
-import { MediaTypeDTO } from 'src/app/types/DTO/media-item-dto';
 import { MediaItem } from 'src/app/types/media-item';
+import { MediaType } from 'src/app/types/media-type';
```

Comment:
It happened again! This is not a big deal, but better to use consistenty for imports

Reply:
Not quite sure what you mean here. In oppose to prefious [sitiatuion](https://github.com/Heliken/themoviedb/pull/5/commits/98f7b74b5a066a7cca4bfc048bfca1eb647a8415#r1173830223) where file was imported from build folder(`/app/..`), here i just got rid of `MediaTypeDTO` as there was no reason to keep both `MediaTypeDTO` and `MediaType` as they are the same. But file was imported from correct folder(`/src/app/....`)

Reply:
I meant you need to choose between two types of imports: relative or absolute. And use chosen type everywhere)

Reply:
Got it, thanks. Maybe you know why VSCode sometimes adds relative import and sometimes - absolute? As i'm not importing manually, all imports are done by IDE, and, as i checked now, they are a lot imports done in either way right now.

Is it okay if i stick to one import type for upcoming features, and will update already existing later on, as there are a lot of them to redo?